### PR TITLE
Spark

### PR DESCRIPTION
**Pull request by AnupT16**

### CODE REVIEW

File: assets/js/theme.js [hunk 1]
```diff
@@ -66,11 +66,12 @@ const hideSearch = () => {
 // CTRL + K -> Open search
 // ESC -> Hide search
 document.addEventListener("keydown", function (e) {
-  if (e.keyCode === 75 && (e.ctrlKey || e.metaKey)) {
+  if (e.key === 75 && (e.ctrlKey || e.metaKey)) {
```
Comment (gouravkhunger) [inline assets/js/theme.js:69]: the object `key` stores [string values](https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values) for keys. `keyCode` was correct, now `Ctrl` / `Command` + `K` doesn't work...

File: assets/js/theme.js [hunk 2]
```diff
@@ -66,11 +66,12 @@ const hideSearch = () => {
...
+    e.preventDefault();
     showSearch();
   }
 
-  if (e.keyCode === 27) {
+  if (e.key === 27) {
```
Comment (gouravkhunger) [inline assets/js/theme.js:74]: Here as well, let's keep it `keyCode`!

**Review by gouravkhunger — CHANGES_REQUESTED** [review]
Hey, left some comments!

**Comment by siddharth-sable** [comment]
Great work

**Pull request merged**